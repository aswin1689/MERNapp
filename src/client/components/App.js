import React from 'react';

type Props = {
    name: number
};

const App = (props: Props) => {
    return <div>Hello {props.name}</div>;
};

export default App;
