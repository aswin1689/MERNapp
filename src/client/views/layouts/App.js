import React, { Component } from 'react';

import ArticlesContainer from '../pages/articles/ArticlesContainer';
import Header from './Header';
import Footer from './Footer';

class App extends Component {
    render() {
        return (
            <React.Fragment>
                <Header title="Blogging MERN" />
                <main>
                    <ArticlesContainer />
                </main>
                <Footer />
            </React.Fragment>
        );
    }
}

export default App;
