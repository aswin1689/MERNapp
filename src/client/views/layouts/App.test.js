import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

test('App should render properly', () => {
    const wrap = shallow(<App name="Aswin" />);

    //expect(wrap.text()).toEqual('Hello Aswin');
    expect(wrap).toMatchSnapshot();
});
