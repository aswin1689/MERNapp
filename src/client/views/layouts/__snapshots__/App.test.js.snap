// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render properly 1`] = `
<React.Fragment>
  <Header
    title="Blogging MERN"
  />
  <main>
    <Connect(ArticlesContainer) />
  </main>
  <Footer />
</React.Fragment>
`;
